certificate-analytics-service/devops/dev/config/application-dev.properties

```diff
@@ -0,0 +1,9 @@
+database.name=certificate_analytics_service
+database.server=localhost
+database.password=cas
+database.username=cas
+spring.activemq.broker-url=tcp://localhost:61616
+spring.activemq.user=activemqUser
+spring.activemq.password=activemqPassword
+hash.salt=salt
+
```

Comment:
Its good if we don't use structure logging locally by default: logging.structured.format.console=